src/main/java/Main.java

```diff
@@ -1,32 +1,33 @@
-import database.RoomDBAccess;
-import database.RoomDBGateway;
-import database.UserDBAccess;
-import database.UserDBGateway;
+import database.*;
+import host_room.*;
+import leave_room.*;
 import signup_and_login.*;
-import ui.UserPresenter;
-import ui.WelcomeView;
+import ui.*;
 
 public class Main {
     private static final String API_URL = "https://api.sheety.co/78ad1edb28469578058ca4c58c3f478b/larklink";
-
-    private static Integer currUserID = 5;
-
     public static void main(String[] args) {
         UserDBGateway userDBAccess = new UserDBAccess(API_URL);
         RoomDBGateway roomDBAccess = new RoomDBAccess(API_URL);
 
-        UserPresenter userPresenter = new UserPresenter(roomDBAccess);
+        UserPresenter userPresenter = new UserPresenter();
         UserInteractor userInteractor = new UserInteractor(userPresenter, userDBAccess);
         UserController userController = new UserController(userInteractor);
         WelcomeView welcomeView = new WelcomeView(userController);
-        welcomeView.prepareGUI();
 
+        LeaveRoomPresenter leaveRoomPresenter = new LeaveRoomPresenter();
```

Comment:
not related to DB stuffs but do you think it is worth keeping left interface right class rule (what we learned in class) here?